test/fetch/getAutoImplementation.ts

```diff
@@ -16,17 +16,20 @@ export const getAutoImplementation = async (
 ): Promise<ResponseWrapper | Record<string, unknown>> => {
   const files = fs.readdirSync(DIR_EXAMPLES);
 
-  const [response] = (
-    await Promise.all(
+  try {
+    return await Promise.any(
```

Comment:
Jo hoeft hier geen `await` bij te stoppen. Hij returned sowieso altijd na het resolven van de Promise.

Reply:
weet ik, is iets in eslint x prettier wat ik nog niet heb opgelost